V1/Brokers/ClickatellBroker.cs

```diff
@@ -18,10 +21,20 @@ public ClickatellBroker(HttpClient httpClient)
 
         public async Task SendSms(object payload)
         {
-            var response = await _httpClient.PostAsJsonAsync(_sendPartialRoot, payload);
+            var response = await _httpClient.PostAsJsonAsync(_sendPartialRoute, payload);
 
             if (!response.IsSuccessStatusCode)
                 throw new ClickatellException(await response.Content.ReadAsStringAsync());
         }
+
+        public async Task<ClickatellResponse> GetBalance()
+        {
+            var response = await _httpClient.GetAsync(_balancePartialRoute);
+
+            if(!response.IsSuccessStatusCode)
+                throw new ClickatellException(await response.Content.ReadAsStringAsync());
+
```

Comment:
Add a private method that does this and just returns the httpResponseMessage. It will avoid writing the line over again